Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'will_paginate', '~> 3.0'
 group :development do
   gem 'awesome_print'
   gem 'spring'
+  gem 'web-console', '~> 2.0'
```

Comment:
This gem seems nice, but this addition should go in another commit, since this is about update to rails 4.2